examples/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+add_subdirectory(krausmaps)
+add_subdirectory(matrices)
+add_subdirectory(numbers)
+add_subdirectory(paulis)
+add_subdirectory(reporters)
+add_subdirectory(superoperators)
```

Comment:
if we do add a `examples/tutorials/main.cpp` file, then above should include:
```
add_subdirectory(tutorials)
```

Reply:
It doesn't because you wanted the executable built in the root directory, and I can't add an executable with no source files even though `target_sources` exists